YAMP.Core/Operators/BinaryOperator.cs

```diff
@@ -48,8 +48,20 @@ public virtual Value Handle(Expression left, Expression right, IDictionary<Strin
 			return Perform(l, r);
 		}
 
-        internal Value PerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping)
+        public Value PerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping)
         {
+            Boolean found;
+            var ret = TryPerformOverFind(left, right, mapping, out found);
+            if (!found)
+                throw new YAMPOperationInvalidException(Op, left, right);
+
+            return ret;
+        }
+
+        public static Value TryPerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping, out Boolean found)
```

Comment:
Is there a reason for turning the usual pattern `Boolean TryX<T>(..., out T value)` around to `T TryX<T>(..., out Boolean success)`?